modules/microshift-low-latency-prepare-workload.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-prepare-workload_{context}"]
+= Preparing a {microshift-short} workload for low latency
+
+To take advantage of low latency, workloads running on {microshift-short} must have the `microshift-low-latency` container runtime configuration set using the `RuntimeClass` feature. The CRI-O `RuntimeClass` object is installed with the `microshift-low-latency` RPM, so only the pod annotations need to be configured.
+
+.Prerequisites
+* You installed the `microshift-low-latency` RPM package.
+* You configured workload partitioning.
+
+.Procedure
+
+. Use the following example to set the following annotations in the pod spec:
++
+[source,text]
+----
+cpu-load-balancing.crio.io: "disable"
+irq-load-balancing.crio.io: "disable"
+cpu-quota.crio.io: "disable"
+cpu-load-balancing.crio.io: "disable"
+cpu-freq-governor.crio.io: "<governor>"
+----
++
+.Example pod that runs `oslat` test:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: oslat
+  annotations:
+    cpu-load-balancing.crio.io: "disable" # <1>
+    irq-load-balancing.crio.io: "disable" # <2>
+    cpu-quota.crio.io: "disable" # <3>
+    cpu-load-balancing.crio.io: "disable" # <4>
+    cpu-freq-governor.crio.io: "<governor>" # <5>
+spec:
+  runtimeClassName: microshift-low-latency # <6>
+  containers:
+  - name: oslat
+    image: quay.io/container-perf-tools/oslat
+    imagePullPolicy: Always
+    resources:
+      requests:
+        memory: "400Mi"
+        cpu: "2"
+      limits:
+        memory: "400Mi"
+        cpu: "2"
+    env:
+    - name: tool
+      value: "oslat"
+    - name: manual
+      value: "n"
+    - name: PRIO
+      value: "1"
+    - name: delay
+      value: "0"
+    - name: RUNTIME_SECONDS
+      value: "60"
+    - name: TRACE_THRESHOLD
+      value: ""
+    - name: EXTRA_ARGS
+      value: ""
+    securityContext:
+      privileged: true
+      capabilities:
+        add:
+          - SYS_NICE
+          - IPC_LOCK
+
+----
+<1> Disables the CPU load balancing for the pod.
+<2> Opts the pod out of interrupt handling (IRQ).
+<3> Disables the CPU completely fair scheduler (CFS) quota at the pod run time.
+<4> Enable or disable C-states for each CPU. Set the value to "disable" to provide the best performance for a high-priority pod. Disabling or enabling CPU load balancing is implemented on the CRI-O level using this field and the `runtimeClassName` field value.
```

Comment:
It's also true for other annotations, they all are implemented by CRIO and all need specific runtimeClassName. If we want to mention that CRI-O is doing the heavylifting here maybe we can do in the section's intro?
```suggestion
<4> Enable or disable C-states for each CPU. Set the value to "disable" to provide the best performance for a high-priority pod.
```